modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -2281,186 +2423,71 @@ bool CvVideoWriter_FFMPEG::open( const char * filename, int fourcc,
     }
 
     /* Lookup codec_id for given fourcc */
-#if LIBAVCODEC_VERSION_INT<((51<<16)+(49<<8)+0)
-    if( (codec_id = codec_get_bmp_id( fourcc )) == CV_CODEC(CODEC_ID_NONE) )
-        return false;
-#else
-    if( (codec_id = av_codec_get_id(fmt->codec_tag, fourcc)) == CV_CODEC(CODEC_ID_NONE) )
+    // codec_get_bmp_id -> removed as deprecated in 2007
+    // https://github.com/FFmpeg/FFmpeg/commit/b3633dee5820cdaa45bf46ebe955b1af59f878a2
+    if ((codec_id = av_codec_get_id(fmt->codec_tag, fourcc)) == CV_CODEC(CODEC_ID_NONE))
     {
-        const struct AVCodecTag * fallback_tags[] = {
+        const struct AVCodecTag* fallback_tags[] = {
 #if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(54, 1, 0)
-// APIchanges:
-// 2012-01-31 - dd6d3b0 - lavf 54.01.0
-//   Add avformat_get_riff_video_tags() and avformat_get_riff_audio_tags().
-                avformat_get_riff_video_tags(),
-#endif
-#if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(55, 25, 100) && defined LIBAVFORMAT_VERSION_MICRO && LIBAVFORMAT_VERSION_MICRO >= 100
-// APIchanges: ffmpeg only
-// 2014-01-19 - 1a193c4 - lavf 55.25.100 - avformat.h
-//   Add avformat_get_mov_video_tags() and avformat_get_mov_audio_tags().
-                avformat_get_mov_video_tags(),
-#endif
-                codec_bmp_tags, // fallback for avformat < 54.1
-                NULL };
-        if( (codec_id = av_codec_get_id(fallback_tags, fourcc)) == CV_CODEC(CODEC_ID_NONE) )
+            // APIchanges:
+            // 2012-01-31 - dd6d3b0 - lavf 54.01.0
+            //   Add avformat_get_riff_video_tags() and avformat_get_riff_audio_tags().
+            avformat_get_riff_video_tags(),
+#endif
+#if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(55, 25, 100) && defined LIBAVFORMAT_VERSION_MICRO \
+    && LIBAVFORMAT_VERSION_MICRO >= 100
+            // APIchanges: ffmpeg only
+            // 2014-01-19 - 1a193c4 - lavf 55.25.100 - avformat.h
+            //   Add avformat_get_mov_video_tags() and avformat_get_mov_audio_tags().
+            avformat_get_mov_video_tags(),
+#endif
+            codec_bmp_tags, // fallback for avformat < 54.1
+            NULL
+        };
+        if ((codec_id = av_codec_get_id(fallback_tags, fourcc)) == CV_CODEC(CODEC_ID_NONE))
         {
             fflush(stdout);
-            fprintf(stderr, "OpenCV: FFMPEG: tag 0x%08x/'%c%c%c%c' is not found (format '%s / %s')'\n",
-                    fourcc, CV_TAG_TO_PRINTABLE_CHAR4(fourcc),
-                    fmt->name, fmt->long_name);
+            fprintf(stderr,
+                    "OpenCV: FFMPEG: tag 0x%08x/'%c%c%c%c' is not found (format '%s / %s')'\n",
+                    fourcc, CV_TAG_TO_PRINTABLE_CHAR4(fourcc), fmt->name, fmt->long_name);
```

Comment:
Please avoid playing with whitespaces.

Patch is huge (>100 lines), so don't make it worse for review.

Reply:
I have enabled autoformat, sorry.